src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -8,6 +8,8 @@ interface ICodeEditorProps extends TextAreaProps {
   value: string;
 }
 
+// TODO: fix firefox/safari bug (NCL-8634)
```

Comment:
Please add also comment mentioning it's a dead code for now that it's not used at all